brian2/codegen/runtime/cython_rt/extension_manager.py

```diff
@@ -131,6 +132,13 @@ def create_extension(self, code, force=False, name=None,
                                          runtime_library_dirs=runtime_library_dirs,
                                          compiler=compiler,
                                          key=key)
+                # Unlock
+                if msvcrt:
+                    msvcrt.locking(f.fileno(), msvcrt.LK_UNLCK,
+                                   os.stat(lock_file).st_size)
+                else:
+                    fcntl.flock(f, fcntl.LOCK_UN)
+            return module
```

Comment:
Did you mean to include these changes in this PR or is this something that was supposed to be in the other one?

Reply:
This change was supposed to fix the build hang that I saw, but it turned out not to work. I'll open a new PR in a sec to use a dedicated module for file locking which seems to correctly deal with the issue.